package.json

```diff
@@ -18,6 +18,8 @@
     "@angularclass/conventions-loader": "1.0.13",
     "@angularclass/hmr": "1.2.2",
     "@angularclass/hmr-loader": "3.0.2",
+    "@ngx-translate/core": "^6.0.1",
```

Comment:
Updated the fixed version but yarn command for `@ngx-translate` has some issues.

```
$ yarn add git@github.com:akveo/ng2-admin.git
yarn add v0.22.0
[1/4] Resolving packages...
error Command failed.
Exit code: 128
Command: git
Arguments: clone github.com:akveo/ng2-admin.git C:\Users\vramamoorthy\AppData\Local\Yarn\cache\v1\.tmp\01867858a5cc6b0b0eed6a441c29c8c6
Directory: D:\Projects\ng2-admin
Output:
Cloning into 'C:\Users\vramamoorthy\AppData\Local\Yarn\cache\v1\.tmp\01867858a5cc6b0b0eed6a441c29c8c6'...
Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
```

```
$ yarn add @ngx-translate/core
yarn add v0.22.0
[1/4] Resolving packages...
error An unexpected error occurred: "https://registry.yarnpkg.com/@ngx-translate/core: Not found".
info If you think this is a bug, please open a bug report with the information provided in "D:\\Projects\\ng2-admin\\yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
``